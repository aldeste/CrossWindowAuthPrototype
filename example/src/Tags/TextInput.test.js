import TextInput from "./TextInput";

describe("TextInput tag", () => {
  it("Should be defined", () => {
    expect(TextInput).toBeDefined();
  });
});
