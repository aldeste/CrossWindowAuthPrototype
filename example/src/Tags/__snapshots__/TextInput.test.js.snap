// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput tag Matches earlier configuration 1`] = `
<input
  className=""
/>
`;
